election.go

```diff
@@ -179,6 +179,46 @@ func (v *vocdoniHandler) votersForElection(msg *apirest.APIdata, ctx *httprouter
 	return ctx.Send(data, http.StatusOK)
 }
 
+func (v *vocdoniHandler) createAndWaitForElection(description *ElectionDescription, census *CensusInfo, profile *FarcasterProfile) (types.HexBytes, error) {
```

Comment:
This is duplicated code, cannot we reuse the existing ones?

Reply:
Is not duplicated, the `createElection` launches the election creation but not waits for the process ends. It is also designed as http handler, not as a helper. I propose that the original handler calls to the helper that i will create